t/pg.t

```diff
@@ -776,6 +776,15 @@ $id = $minion->enqueue(test => [] => {parents => [-1]});
 $job = $worker->dequeue(0);
 is $job->id, $id, 'right id';
 ok $job->finish, 'job finished';
+$id = $minion->enqueue(test => [] => {parents => [-1]});
```

Comment:
Using negative numbers as ids seems wrong, what's the reasoning?

Reply:
Copied it from existing test code (line 775)

Reply:
That's weird.